### ISSUE TITLE
[high] Keyboard Navigation: Chat: Cannot focus the chat message action buttons, visible on hover only

### ISSUE THREAD

**Issue by ogumen** [Expensify/App#5674]
## Action Performed:
Prerequisites:
- User has created personal account and is logged into same.
- User has concierge chat history.

1. Using Chrome, open https://staging.new.expensify.com/
2. Tab over to the Concierge chat in the chat history and press Enter.
3. Hover the mouse pointer over previous chat messages in the history and note the context icon buttons that appear. Edit, mark as unread, etc.
4. Try using any keyboard inputs to access the buttons/menu for any of the messages.

## Expected Result:
Keyboard users must have access to all the features available to other users. Users should be able to tab over to the context buttons for each contact.

## Actual Result:
Since the message action buttons are only displayed on hovering the mouse they cannot be focused using only the KB to navigate. There is a right-click context menu with the same functions that can be accessed by pressing the windows context menu key on the keyboard, but users are not made of aware of this functionality. It must be discovered.

HTML
`<div class="css-1dbjc4n r-u8s1d" style="right: 16px; top: -16px; z-index: 1;"><div class="css-1dbjc4n r-18u37iz" style="background-color: rgb(255, 255, 255); border-color: rgb(236, 236, 236); border-radius: 8px; border-width: 1px;"><div class="css-1dbjc4n"><div class="css-1dbjc4n"><div aria-label="Copy to Clipboard" tabindex="0" class="css-1dbjc4n r-1loqt21 r-1otgn73" style="border-radius: 4px; margin: 4px; padding: 4px;"><svg xmlns="http://www.w3.org/2000/svg"
viewBox="0 0 16 16" width="20" height="20" fill="#C6C9CA"><path d="M6 4h4c.3 0 .5-.2.5-.5v-1c0-.3-.2-.5-.5-.5H9c0-.6-.4-1-1-1s-1 .4-1 1H6c-.3 0-.5.2-.5.5v1c0 .3.2.5.5.5zm2-2.5c.1 0 .2.1.2.2S8.1 2 8 2s-.2-.1-.2-.2.1-.3.2-.3z"></path><path d="M12 2.5h-.5v1.6c0 .6-.4 1-1 1h-5c-.6 0-1-.4-1-1V2.5H4c-.6 0-1 .4-1 1v10c0 .6.4 1 1 1h8c.6 0 1-.4 1-1v-10c0-.6-.4-1-1-1z"></path></svg></div></div></div><div class="css-1dbjc4n"><div class="css-1dbjc4n">...</div>`

## Workaround:
No workaround

## Area issue was found in:
Chat page

## Failed WCAG checkpoints
2.1.1

## User impact:
Keyboard users will not be able to access the action buttons when they are not visible.

## Suggested resolution:
Have the context button appear on focus, not hover so they are in the tab order of the keyboard. Alternatively provide visible textual instructions on how a user can access the context menu. Have the text encountered before the chat messages in the Tab/reading order so that screen readers encounter it before the instructions are needed.
https://www.w3.org/WAI/WCAG21/Techniques/general/G202.html

## Platform:
Where is this issue occurring?

- Web

**Version Number:**
**Reproducible in staging?:**Yes
**Reproducible in production?:**
**Notes/Photos/Videos:** Any additional supporting documentation
**Issue reported by:**

https://user-images.githubusercontent.com/88733897/136203696-8b60c06a-8d43-4ca8-913a-600a7bf7e7f5.mp4

![Bug5187521_Keyboard_Navigation-Chat-Cannot_focus_on_chat_message_action_buttons__visible_on_hover_only](https://user-images.githubusercontent.com/88733897/136203706-d857f9dc-2a32-40b7-ba1d-96354c8a2393.png)

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)
